server/test/routes/likes.test.js

```diff
@@ -0,0 +1,329 @@
+
+import { expect, use, request } from 'chai';
+import { describe, it, before } from 'mocha';
+import chaiHttp from 'chai-http';
+import server from '../../index';
+
+use(chaiHttp);
+
+let token;
+let slug;
+let likeRoute;
+
+
+describe('Testing likes routes /api/auth/:slug/like', () => {
+  // Sign up user to the database
+  before((done) => {
+    request(server)
+      .post('/api/users')
+      .send({
+        user: {
+          firstName: 'Chidiebere',
+          lastName: 'TheGreat',
+          username: 'username101',
+          password: 'R!vend1elLl121//',
+          email: 'username1e12@gmail.com',
+        },
+      })
+      .end((err, res) => {
+        const temp = res.body.user.token;
+        token = temp;
+        expect(temp).to.equal(token);
+        done();
+      });
+  });
+  // Create an article
+  before((done) => {
+    request(server)
+      .post('/api/articles')
+      .set('Authorization', token)
+      .send({
+        article: {
+          title: 'Building a balance in life',
+          description: 'Talks about the beautify  of family life',
+          body: 'there is need for us to go home to our '
+        + 'family as quick as we can. It is one way to keep ourselves happy and fulfiled',
+        },
+      })
+      .end((err, res) => {
+        const temp = res.body.article.slug;
+        slug = temp;
+        likeRoute = `/api/articles/${slug}/like`;
+
+        done();
+      });
+  });
+
+  describe('when like informations are retrieved when no one has liked the article', () => {
+    it('should return a status code of 204 and a data property', (done) => {
+      request(server)
+        .get(likeRoute)
+        .set('Authorization', token)
+        .end((err, res) => {
+          expect(res)
+            .property('status').to.equal(204);
+          done();
+        });
+    });
+  });
+  describe('when a user likes an aricle via POST /api/auth/:slug/like', () => {
+    describe('when authentication is not provided ', () => {
+      it('should return a 401(Unauthorized', (done) => {
+        request(server)
+          .post(likeRoute)
+          .end((err, res) => {
+            expect(res)
+              .property('status').to.equal(401);
+            done();
+          });
+      });
+      it('should return an error object in the body', (done) => {
```

Comment:
there is no need for this extra `it`, you can check your assertions up in the first `it` block.